src/jobManager.ts

```diff
@@ -109,9 +111,7 @@ async function processJob(jobData: any) {
 function failAfterMaxJobTime(priority: string, receiptHandle: string, job: any) {
   return setTimeout(async () => {
     try {
-      sqs.sendMessage(job.id, BUILD.JOB_STATES.ERRORED, {
-        turtleVersion: job.config.turtleVersion,
-      });
+      sqs.sendMessage(job.id, BUILD.JOB_STATES.ERRORED, { turtleVersion });
```

Comment:
Why do you send `turtleVersion` to www only if the build has failed? I guess it'd be probably better to send this information with `IN_PROGRESS` status (few lines below).

Reply:
it's sent on finished and on error